docs/public/static/schemas/unversioned/eas-json-build-android-schema.js

```diff
@@ -50,8 +50,8 @@ export default [
     enum: ['app-bundle', 'apk'],
     description: [
       'Type of the artifact you want to build. It controls which Gradle task will be used to build the project. It can be overridden by `gradleCommand` or `developmentClient: true` option.',
-      ' - `app-bundle` - `:app:bundleRelease`',
-      ' - `apk` - `:app:assembleRelease`',
+      ' - `app-bundle` - `:app:bundleRelease` (creates `.aab` artifact)',
+      ' - `apk` - `:app:assembleRelease` (creates `.apk` artifact)',
```

Comment:
```suggestion
      ' - `app-bundle` - `:app:bundleRelease` (creates **.aab** artifact)',
      ' - `apk` - `:app:assembleRelease` (creates **.apk** artifact)',
```

Let's follow our writing style guide to use [bold names for files](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text). These artifacts are considered files too.

This will result in:
![CleanShot 2024-04-30 at 18 39 52@2x](https://github.com/expo/expo/assets/10234615/c5645196-6976-4e54-b1cd-515244ba3255)